ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -196,8 +211,10 @@ class LargeTitleView: UIView {
 
         avatarView.centerYAnchor.constraint(equalTo: contentStackView.centerYAnchor).isActive = true
 
-        // title button setup
-        contentStackView.addArrangedSubview(titleButton)
+        // title container setup
```

Comment:
Super tiny nit - I don't think this comment is useful, consider removing.

Reply:
Fair.